mtp_cashbook/templates/cashbook/transactions_history.html

```diff
@@ -77,11 +104,11 @@ <h3 class="HistoryHeader {% if transaction_status == 'uncredited' %}HistoryHeade
             <a class="u-nojs-hidden print-hidden HistoryHeader-aside ShowHide js-ShowHide" href="#HistoryBatch-{{ forloop.parentloop.counter }}-{{ forloop.counter }}">
               <span class="ShowHide-hide">
                 {% trans 'Collapse' %}
-                <span class="ShowHide-icon">&minus;</span>
+                <span class="ShowHide-icon">&ndash;</span>
               </span>
               <span class="ShowHide-show">
                 {% trans 'Collapsed' %}
-                <span class="ShowHide-icon">&plus;</span>
+                <span class="ShowHide-icon">+</span>
```

Comment:
Shouldn't these be images with appropriate `alt`s? (The en dash, I think, won't be read out by screen readers meaningfully, for example?)

Also… ultimately, they should be added by JS because they don't do anything without it.